include/mbedtls/ssl_ticket.h

```diff
@@ -109,6 +109,33 @@ int mbedtls_ssl_ticket_setup( mbedtls_ssl_ticket_context *ctx,
     mbedtls_cipher_type_t cipher,
     uint32_t lifetime );
 
+/**
+ * \brief           Rotate session ticket key to new specified key.
+ *                  Provides for external control of session ticket
+ *                  encryption key rotation.
+ *
+ * \param ctx       Context to be set up
+ * \param name      Session ticket encryption key name
+ * \param nlen      Session ticket encryption key name len
+ * \param k         Session ticket encryption key
+ * \param klen      Session ticket encryption key length
+ * \param lifetime  Tickets lifetime in seconds
+ *                  Recommended value: 86400 (one day).
+ *
+ * \note            \c nlen must match sizeof( ctx->name )
```

Comment:
Thanks for documenting that. Perhaps also mention that `klen` must not be less than the previous value?